update/changelog.go

```diff
@@ -0,0 +1,31 @@
+package update
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	githubClient "github.com/google/go-github/v47/github"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/services/github"
+)
+
+func printRelease(release githubClient.RepositoryRelease) {
+	fmt.Printf("Changelog of the version %v\n\n", release.GetTagName())
+	fmt.Printf("%v\n\n", strings.ReplaceAll(*release.Body, "\\r\\n", "\r\n"))
+}
+
+func ShowLastChangelog() error {
+	ctx := context.Background()
+
+	client := github.NewClient()
+
+	cliLastRelease, err := client.GetLatestRelease(ctx)
+	if err != nil {
+		return errgo.Notef(err, "fail to get last CLI release")
+	}
+	printRelease(*cliLastRelease)
```

Comment:
Should we check if cliLastRelease is nil ?

Reply:
`cliLastRelease` is always defined if there is no error. So it should be safe :)

Reply:
It might break in an update but ok it could be nitpicking :)